dev-tools/reconfigurator-cli/src/main.rs

```diff
@@ -380,8 +430,12 @@ enum Commands {
     Save(SaveArgs),
     /// load state from a file
     Load(LoadArgs),
+    /// generate and load an example system
+    LoadExample(LoadExampleArgs),
     /// show information about what's in a saved file
     FileContents(FileContentsArgs),
+    /// reset the state of the REPL
+    Reset,
```

Comment:
Can we give this a more descriptive name, like maybe `clear` or `wipe`?

When you say "reset the REPL" I think we're talking about terminal settings or something.  What we're doing here is wiping all the in-memory state that describes the underlying system and making it match what you would get if you started the program again, right?

Reply:
Changed this to "wipe".